com.microsoft.Edge.yaml

```diff
@@ -0,0 +1,155 @@
+app-id: com.microsoft.Edge
+# desktop-file-name-prefix: '(Testing) '
```

Comment:
Why leave commented line around?

Reply:
Because I wanted to check if everything worked before I remove them. I completely forgot to remove them so I'll do it now.

Reply:
https://github.com/flathub/flathub/pull/1992/commits/c1e6de6049a0d4737cfb51970855713d90a679d8